ssg/ext/packaging/requirements.py

```diff
@@ -0,0 +1,160 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+from __future__ import absolute_import, division, print_function
+
+import re
+import string
+import sys
+
+from pyparsing import (  # noqa: N817
```

Comment:
When I build on RHEL 8.6 inside https://src.fedoraproject.org/tests/scap-security-guide/pull-request/3# I get:

```
Traceback (most recent call last):
  File "/tmp/tmp.YdR9bYfG6P/content/build-scripts/compile_all.py", line 9, in <module>
    import ssg.build_profile
  File "/tmp/tmp.YdR9bYfG6P/content/ssg/build_profile.py", line 7, in <module>
    from .build_yaml import ProfileWithInlinePolicies
  File "/tmp/tmp.YdR9bYfG6P/content/ssg/build_yaml.py", line 18, in <module>
    from .build_cpe import CPEDoesNotExist, CPEALLogicalTest, CPEALFactRef, CPEItem, ProductCPEs
  File "/tmp/tmp.YdR9bYfG6P/content/ssg/build_cpe.py", line 16, in <module>
    from .boolean_expression import Algebra, Symbol, Function
  File "/tmp/tmp.YdR9bYfG6P/content/ssg/boolean_expression.py", line 2, in <module>
    from .ext.packaging import requirements, version
  File "/tmp/tmp.YdR9bYfG6P/content/ssg/ext/packaging/requirements.py", line 10, in <module>
    from pyparsing import (  # noqa: N817
ModuleNotFoundError: No module named 'pyparsing'
make[2]: *** [rhel8/CMakeFiles/rhel8-compile-all.dir/build.make:74: rhel8/profiles] Error 1
make[1]: *** [CMakeFiles/Makefile2:1142: rhel8/CMakeFiles/rhel8-compile-all.dir/all] Error 2
make: *** [Makefile:154: all] Error 2
```
I think you will have to add pyparsing to the list of dependencies.

Reply:
Yeah, yeah, working on it.

Reply:
yes. it's `python3-pyparsing` on RHEL 8

Reply:
Fixed